resources/user-agents/browsers/mobile-safari/mobile-safari-15-x.json

```diff
@@ -24,7 +24,7 @@
             "iPod": "iPod Touch"
           },
           "platforms": [
-            "iOS_A_15_2", "iOS_A_15_1", "iOS_A_15_0",
+            "iOS_A_15_4", "iOS_A_15_3", "iOS_A_15_2", "iOS_A_15_1", "iOS_A_15_0",
```

Comment:
This results in the error : 
`Browscap ini file could not be built in UserAgentsTest\V4\FullTest test class, there was an uncaught exception: OutOfBoundsException (Platform "iOS_A_15_4" does not exist in data)`

Please check if these platforms are set in platforms.json file

Reply:
Interesting, I do not get this error when running